src/crate/testing/layer.py

```diff
@@ -155,7 +155,8 @@ def __init__(self,
                  cluster_name=None,
                  host="127.0.0.1",
                  settings=None,
-                 verbose=False):
+                 verbose=False,
+                 env={}):
```

Comment:
Please use `env=None`. Default values are bound at method definition time, not at evaluation time.

```
>>> def foo(x={}):
...     x[random.randint(0, 10)] = 1
...     print(x)
... 
>>> foo()
{4: 1}
>>> foo()
{4: 1, 0: 1}
>>> foo()
{4: 1, 0: 1, 7: 1}
```